<?xml version="1.0" encoding="utf-8"?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from model.xacro                    | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<sdf version="1.5" xmlns:xacro="http://www.ros.org/wiki/xacro">
  <!-- properties width, model, and material must be defined before including this file -->
  <model name="ring_1">
    <link name="hanoi_ring_1">
      <velocity_decay>
        <linear>0.01</linear>
        <angular>1.0</angular>
      </velocity_decay>
      <visual name="part_T">
        <pose>0 0.02375 0.02 0 0 0</pose>
        <geometry>
          <box>
            <size>0.07 0.0225 0.04</size>
          </box>
        </geometry>
        <material>
          <script>
            <name>Gazebo/Red</name>
          </script>
        </material>
      </visual>
      <collision name="part_T">
        <pose>0 0.02375 0.02 0 0 0</pose>
        <geometry>
          <box>
            <size>0.07 0.0225 0.04</size>
          </box>
        </geometry>
        <surface>
          <bounce>
            <restitution_coefficient>0.0</restitution_coefficient>
          </bounce>
          <friction>
            <ode>
              <mu>1</mu>
              <mu2>1</mu2>
            </ode>
          </friction>
          <contact>
            <ode>
              <max_vel>0.01</max_vel>
              <min_depth>0.001</min_depth>
            </ode>
          </contact>
        </surface>
      </collision>
      <visual name="part_B">
        <pose>0 -0.02375 0.02 0 0 0</pose>
        <geometry>
          <box>
            <size>0.07 0.0225 0.04</size>
          </box>
        </geometry>
        <material>
          <script>
            <name>Gazebo/Red</name>
          </script>
        </material>
      </visual>
      <collision name="part_B">
        <pose>0 -0.02375 0.02 0 0 0</pose>
        <geometry>
          <box>
            <size>0.07 0.0225 0.04</size>
          </box>
        </geometry>
        <surface>
          <bounce>
            <restitution_coefficient>0.0</restitution_coefficient>
          </bounce>
          <friction>
            <ode>
              <mu>1</mu>
              <mu2>1</mu2>
            </ode>
          </friction>
          <contact>
            <ode>
              <max_vel>0.01</max_vel>
              <min_depth>0.001</min_depth>
            </ode>
          </contact>
        </surface>
      </collision>
      <visual name="part_L">
        <pose>-0.02375 0 0.02 0 0 0</pose>
        <geometry>
          <box>
            <size>0.0225 0.025 0.04</size>
          </box>
        </geometry>
        <material>
          <script>
            <name>Gazebo/Red</name>
          </script>
        </material>
      </visual>
      <collision name="part_L">
        <pose>-0.02375 0 0.02 0 0 0</pose>
        <geometry>
          <box>
            <size>0.0225 0.025 0.04</size>
          </box>
        </geometry>
        <surface>
          <bounce>
            <restitution_coefficient>0.0</restitution_coefficient>
          </bounce>
          <friction>
            <ode>
              <mu>1</mu>
              <mu2>1</mu2>
            </ode>
          </friction>
          <contact>
            <ode>
              <max_vel>0.01</max_vel>
              <min_depth>0.001</min_depth>
            </ode>
          </contact>
        </surface>
      </collision>
      <visual name="part_R">
        <pose>0.02375 0 0.02 0 0 0</pose>
        <geometry>
          <box>
            <size>0.0225 0.025 0.04</size>
          </box>
        </geometry>
        <material>
          <script>
            <name>Gazebo/Red</name>
          </script>
        </material>
      </visual>
      <collision name="part_R">
        <pose>0.02375 0 0.02 0 0 0</pose>
        <geometry>
          <box>
            <size>0.0225 0.025 0.04</size>
          </box>
        </geometry>
        <surface>
          <bounce>
            <restitution_coefficient>0.0</restitution_coefficient>
          </bounce>
          <friction>
            <ode>
              <mu>1</mu>
              <mu2>1</mu2>
            </ode>
          </friction>
          <contact>
            <ode>
              <max_vel>0.01</max_vel>
              <min_depth>0.001</min_depth>
            </ode>
          </contact>
        </surface>
      </collision>
      <visual name="part_H">
        <pose>0 0.05 0.02 0 0 0</pose>
        <geometry>
          <box>
            <size>0.03 0.03 0.04</size>
          </box>
        </geometry>
        <material>
          <script>
            <name>Gazebo/Red</name>
          </script>
        </material>
      </visual>
      <collision name="part_H">
        <pose>0 0.05 0.02 0 0 0</pose>
        <geometry>
          <box>
            <size>0.03 0.03 0.04</size>
          </box>
        </geometry>
        <surface>
          <bounce>
            <restitution_coefficient>0.0</restitution_coefficient>
          </bounce>
          <friction>
            <ode>
              <mu>1</mu>
              <mu2>1</mu2>
            </ode>
          </friction>
          <contact>
            <ode>
              <max_vel>0.01</max_vel>
              <min_depth>0.001</min_depth>
            </ode>
          </contact>
        </surface>
      </collision>
      <!-- the handle part -->
      <inertial name="inertia">
        <pose>0 0 0.02 0 0 0</pose>
        <mass>0.02</mass>
        <inertia>
          <ixx>3.02916666667e-05</ixx>
          <iyy>3.02916666667e-05</iyy>
          <izz>5.525e-05</izz>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyz>0</iyz>
        </inertia>
      </inertial>
    </link>
  </model>
</sdf>

